lib/full-site-editing/full-site-editing.php

```diff
@@ -11,7 +11,8 @@
  * @return boolean Whether the current theme is an FSE theme or not.
  */
 function gutenberg_is_fse_theme() {
```

Comment:
Maybe some confusion is stemming from the fact that we are using this function to gate FSE features/functionality in different places. Would `gutenberg_is_fse_active` with a filterable return be a more appropriate name? It could even wrap the existing `gutenberg_is_fse_theme` function and allow for more complicated checks in the future.